experimental/GroebnerWalk/benchmark/benchmarks.jl

```diff
@@ -0,0 +1,60 @@
+using Oscar
+using GroebnerWalk
```

Comment:
this won't work anymore
(and there are many more occurrences of `using GroebnerWalk` in your PR that need to be adapted)